lib/cocoapods/command/deintegrate.rb

```diff
@@ -1,5 +1,7 @@
 module Pod
   class Command
+    # @since 1.0.0.beta.1
```

Comment:
It's just a string - I'll see if there's a more appropriate annotation, but I like this idea.

Also, YARD allows for multiple of any tag.